### PR TITLE
Update WikiFur.com rules

### PR DESCRIPTION
**Pull request by Nintendofan885**
Update WikiFur.com rules to redirect subdomains that don't start with two letter

==List of subdomains (all support https)==
wikifur.com
admin.wikifur.com
be.wikifur,com
be-tarask.wikifur.com
board.wikifur.com
by.wikifur.com
cs.wikifur.com
da.wikifur.com
de.wikifur.com
dk.wikifur.com
dumps.wikifur.com
en.wikifur.com
eo.wikifur.com
es.wikifur.com
et.wikifur.com
fr.wikifur.com
fursuit.wikifur.com
hu.wikifur.com
it.wikifur.com
ja.wikifur.com
ko.wikifur.com
nb.wikifur.com
nl.wikifur.com
no.wikifur.com
ny.wikifur.com
pl.wikifur.com
pt.wikifur.com
pool.wikifur.com
ru.wikifur.com
se.wikifur.com
stats.wikifur.com
sv.wikifur.com
tl.wikifur.com
uk.wikifur.com
www.wikifur.com
zh.wikifur.com

### CODE REVIEW

File: src/chrome/content/rules/WikiFur.xml [hunk 1]
```diff
@@ -1,8 +1,12 @@
 <ruleset name="WikiFur">
   <target host="wikifur.com" />
   <target host="*.wikifur.com" />
```
Comment (pipboy96) [inline src/chrome/content/rules/WikiFur.xml:3]: Please avoid using wildcards and enumerate subdomains using domain enumeration tools, such as [Amass](https://github.com/OWASP/Amass).

```suggestion
	<target host="www.wikifur.com" />
	<target host="be.wikifur.com" />
	<target host="be-tarask.wikifur.com" />
	<target host="board.wikifur.com" />
	<target host="by.wikifur.com" />
	<target host="cs.wikifur.com" />
	<target host="da.wikifur.com" />
	<target host="de.wikifur.com" />
	<target host="dk.wikifur.com" />
	<target host="dumps.wikifur.com" />
	<target host="en.wikifur.com" />
	<target host="eo.wikifur.com" />
	<target host="es.wikifur.com" />
	<target host="et.wikifur.com" />
	<target host="fr.wikifur.com" />
	<target host="fursuit.wikifur.com" />
	<target host="hu.wikifur.com" />
	<target host="it.wikifur.com" />
	<target host="ja.wikifur.com" />
	<target host="ko.wikifur.com" />
	<target host="nb.wikifur.com" />
	<target host="nl.wikifur.com" />
	<target host="no.wikifur.com" />
	<target host="ny.wikifur.com" />
	<target host="pl.wikifur.com" />
	<target host="pool.wikifur.com" />
	<target host="pt.wikifur.com" />
	<target host="ru.wikifur.com" />
	<target host="stats.wikifur.com" />
	<target host="tl.wikifur.com" />
	<target host="uk.wikifur.com" />
	<target host="zh.wikifur.com" />
```

File: src/chrome/content/rules/WikiFur.xml [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
-
-  <rule from="^http://([a-z][a-z]\.)?wikifur\.com/" to="https://$1wikifur.com/" />
+  
+  <securecookie host=".+" name=".+"/>
+  <rule from="^http:" to="https:" />
+  
+  <test url="http://en.wikifur.com/" />
+  <test url="http://www.wikifur.com/" />
+  <test url="http://fr.wikifur.com/" />
+  <test url="http://fursuit.wikifur.com/" />
```
Comment (pipboy96) [inline src/chrome/content/rules/WikiFur.xml:11]: These tests will no longer be needed if you list subdomains individually.

```suggestion
```

**Review by pipboy96 — CHANGES_REQUESTED** [review]
Please also use tab characters instead of two spaces for indentation.

**Review by pipboy96 — APPROVED** [review]

File: src/chrome/content/rules/WikiFur.xml [hunk 3]
```diff
@@ -1,8 +1,38 @@
 <ruleset name="WikiFur">
   <target host="wikifur.com" />
```
Comment (cschanaj) [inline src/chrome/content/rules/WikiFur.xml:2]:
```suggestion
	<target host="wikifur.com" />
```

File: src/chrome/content/rules/WikiFur.xml [hunk 4]
```diff
@@ -1,8 +1,38 @@
...
+	<target host="pool.wikifur.com" />
+	<target host="pt.wikifur.com" />
+	<target host="ru.wikifur.com" />
+	<target host="stats.wikifur.com" />
+	<target host="tl.wikifur.com" />
+	<target host="uk.wikifur.com" />
+	<target host="zh.wikifur.com" />
+  
+  <securecookie host=".+" name=".+"/>
+  <rule from="^http:" to="https:" />
```
Comment (cschanaj) [inline src/chrome/content/rules/WikiFur.xml:37]:
```suggestion
	<securecookie host=".+" name=".+"/>
	<rule from="^http:" to="https:" />
```

**Review by cschanaj — CHANGES_REQUESTED** [review]
Thank you for your PR. Would you apply the suggestions and use tabs for indentations as per our [contributing guide](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#indentation--misc-stylistic-conventions)?

**Comment by Nintendofan885** [comment]
Thank you

**Pull request merged**